packages/electrode-archetype-react-component/config/webpack/partial/styles.js

```diff
@@ -10,6 +10,11 @@ const cssnext = archDevRequire("postcss-cssnext");
 const styleLoader = archDevRequire.resolve("style-loader");
 const cssLoader = archDevRequire.resolve("css-loader");
 const postcssLoader = archDevRequire.resolve("postcss-loader");
+const stylusLoader = archDevRequire.resolve("stylus-relative-loader");
+
+const configPath = Path.resolve("config.json");
```

Comment:
We don't currently have any configuration for components like we do apps, AFAICT. `config.json` is probably not what we want to do, but I want to get some feedback before implementing anything more complicated.